src/fragments/lib/graphqlapi/js/working-with-files.mdx

```diff
@@ -0,0 +1,910 @@
+The Storage and GraphQL API categories can be used together to associate a file, such as an image or video, with a particular record. For example, you might create a `User` model with a profile picture, or a `Post` model with an associated image. With Amplify's GraphQL API and Storage categories, you can reference the file within the model itself to create an association.
+
+To get started, run the following in an existing Amplify project:
+
+```bash
+npm i @aws-amplify/ui-react  #For Authenticator component
+amplify add auth             #Select default configuration
+amplify add storage          #Select S3 with read/write access
+amplify add api              #Select Cognito User Pool for authorization type
+```
+
+When prompted, use the following schema:
+
+```graphql
+type Song @model @auth(rules: [{ allow: public}]) {
+  id: ID!
+  name: String!
+  covertArtKey: String #Set as optional to allow adding file after initial create
+}
+```
+
+Save the schema and run `amplify push` to deploy the changes.
+
+To use files and attachments you need authorization credentials for reading and writing to Amazon S3, which `amplify add auth` configures in the default setting along with a Cognito User Pool.
```

Comment:
I would pull this up right after the CLI commands - I think it would flow better that way

Reply:
Note: the wording here has been updated (see Ivan's suggestion), which is why this is marked as "outdated" - however, the placement is the same.

Reply:
LGTM